tx-pool/src/pool.rs

```diff
@@ -323,18 +289,7 @@ impl TxPool {
 
     pub(crate) fn remove_tx(&mut self, id: &ProposalShortId) -> bool {
         let entries = self.pool_map.remove_entry_and_descendants(id);
-        if !entries.is_empty() {
-            for entry in entries {
-                self.update_statics_for_remove_tx(entry.size, entry.cycles);
-            }
-            return true;
-        }
-
-        if let Some(entry) = self.pool_map.remove_entry(id) {
-            self.update_statics_for_remove_tx(entry.size, entry.cycles);
-            return true;
-        }
-        false
+        !entries.is_empty()
```

Comment:
I believe this change makes the logic inconsistent with the previous commit.

Reply:
`self.pool_map.remove_entry(id)` will never return `Some(_)`, because the above `self.pool_map.remove_entry_and_descendants(id)` already removed the entry with `id`.